packages/database/src/modelClasses/Entity.js

```diff
@@ -367,6 +367,17 @@ export class EntityModel extends MaterializedViewLogDatabaseModel {
     );
   }
 
+  async updateEntityAttributes(code, attributes) {
```

Comment:
Do we need to get the existing attributes on the entity and merge them here?

Reply:
Had a conversation with Andrew, Juliana, and Georgia about this, and we concluded that we will replace the attributes if they are provided:
<img width="1012" alt="Screenshot 2024-08-06 at 12 44 31 pm" src="https://github.com/user-attachments/assets/d734a71a-7926-4bfc-a27d-7aa7b2efcd07">